src/box/memtx_tx.h

```diff
@@ -572,6 +627,8 @@ memtx_tx_snapshot_clarify(struct memtx_tx_snapshot_cleaner *cleaner,
 
 /**
  * Free resources.in shapshot @cleaner.
+ *
```

Comment:
Since we removed garbage collection in this method we should remove the comment as well.

Reply:
Sorry, wasn't careful enough, fixed.